config/locales/en.yml

```
@@ -483,6 +483,18 @@ en:
     dropdown_default_text: Select a question to start your route from
     legend_hint_text: A route can only start from a question where people select one item from a list
     legend_text: Which question do you want your route to start from?
+    no_remaining_routes: A route can only start from a question where people select one item from a list. You can only add one route per question.
+    no_remaining_routes_heading: You have no more questions to start a route from
+    routing_requirements_not_met_heading: What you need to create a route
+    routing_requirements_not_met_html: |
+      <p>Before you can add a route you'll need:</p>
```

Can use a curly quote here:
```suggestion
      <p>Before you can add a route you’ll need:</p>
```